### PR TITLE
 #192 Add experiments

### LINKED ISSUE #192: As a developer, I want to propose experiments for users to test
Experiments are features that are either:

- under development
- in an early stage

I want a simple way to propose to the users a way to enable/disable experiments. The users will give me feedback on the feature, or/and I can put telemetry on it, so I know if I continue developing it, if the feature have bugs to fix.

### PR DESCRIPTION
**Pull request by MichaelBailly**
Hello there,
this is a pull-request to add a way for developers to push new, and maybe unfinished, features to the users. I called it "experiments". The objective is that it should be simple and quick for developers to add a new experiment. That's why the experiments state is not persisted on the server, and so experiments array is hard-coded in the SPA.

- the user can enable/disable experiment. It's enabled/disabled for the current web browser only
- it relies on the angular-feature-flags component that we already use
- it's persisted in the localStorage

Screenshot_20201012_112247

Screenshot_20201012_112432

Closes #192

### CODE REVIEW

File: src/linagora.esn.unifiedinbox/i18n/en.json [hunk 1]
```diff
@@ -203,6 +207,7 @@
   "Test": "Test",
   "Thanks a lot for your message. I am currently out of office and will get back to you soon.": "Thanks a lot for your message. I am currently out of office and will get back to you soon.",
   "then": "then",
+  "There is no current experiments. Come back later !": "There is no current experiments. Come back later !",
```
Comment (rezk2ll) [inline src/linagora.esn.unifiedinbox/i18n/en.json:210]:
```suggestion
  "There are no current experiments. Come back later!": "There are no current experiments. Come back later!",
```

**Review by tuanlc — COMMENTED** [review]
This cool feature should not limit to only inbox module, don't you think?

File: src/linagora.esn.unifiedinbox/i18n/en.json (hunk 1 above)
Comment (MichaelBailly) [reply src/linagora.esn.unifiedinbox/i18n/en.json:210]: @rezk2ll you're right this english is not good. What do you think of `No experiment is currently available. Come back later!` ?

**Comment by MichaelBailly** [comment]
> This cool feature should not limit to only inbox module, don't you think?

I let other FT extract the code to a generic one then, when they'll need it. And if they never need it, the code will stay in Inbox and not pollute the other SPA.

File: src/linagora.esn.unifiedinbox/i18n/en.json (hunk 1 above)
Comment (rezk2ll) [reply src/linagora.esn.unifiedinbox/i18n/en.json:210]: sounds good.

**Review by rezk2ll — APPROVED** [review]

**Review by tuanlc — APPROVED** [review]

**Review by huy-ta — APPROVED** [review]

**Pull request merged**